src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -379,6 +420,16 @@ const MutationCollapsible = ({
           }}
         />
       ) : undefined}
+      {isConvertingToVus ? (
+        <AddVusModal
+          hugoSymbol={hugoSymbol}
+          isGermline={isGermline}
+          vusList={vusData}
+          onCancel={() => setIsConvertingToVus(false)}
+          onConfirm={handleDemoteToVus}
+          convertOptions={{ initialAlterations: mutationName.split(', '), isConverting: true, mutationUuid }}
```

Comment:
Maybe true for newly added, but not true for existing string mutation
```suggestion
          convertOptions={{ initialAlterations: mutationName.split(','), isConverting: true, mutationUuid }}
```